tests/plans/test_reflectometry.py

```diff
@@ -60,3 +79,467 @@ def test_refl_adaptive_scan_creates_refl_param_device_and_dae(RE):
         assert isinstance(scan.call_args[1]["dae"], SimpleDae)
         assert isinstance(scan.call_args[1]["block"], ReflParameter)
         assert scan.call_args[1]["block"].name == param_name
+
+
+# Auto-Alignment Utils
+
+
+@pytest.fixture
+async def dae():
```

Comment:
https://github.com/ISISComputingGroup/ibex_bluesky_core/blob/main/tests/conftest.py#L23 could use this instead